HOWTO.md

```diff
@@ -343,11 +343,41 @@ From inside NativeScript the code looks like this:
    }
 ```
 
-## Potential Gotchas and Issues
+## The NativeScriptEmbedderDelegate protocol
+NativeScript 4.2.0 gives embedders more control over the NativeScript application using the *NativeScriptEmbedder* class and the *NativeScriptEmbedderDelegate* protocol. In order to take advantage of them you need to do few additional steps:
 
-1. The embedded NativeScript view is currently full screen; there is no easy way to currently embed it as a sub-view of your view.  To embed it as a sub-view would require you to modify the application.js, page.js, and frame.js classes inside NativeScript.  Not something that would be quick and easy.  It is possible, but would take a bit of work.
+1. Add *tns-core-modules* **src** folder to your **Header Search Paths**:
+
+```
+$(PROJECT_DIR)/../nsapp/node_modules/tns-core-modules/platforms/ios/src
+```
+2. Add  **NativeScriptEmbedder.m** file to **Compile Sources**(*Build Phases -> Compile Sources*)
+
+ ![docs/nativescript_embedder.PNG](docs/nativescript_embedder.PNG)
+ 
+3. Update the class you want to get a reference to the NativeScript app *rootViewController* to conform to the **NativeScriptEmbedderDelegate** protocol. For example:
+
+```
+@interface ViewController () <NativeScriptEmbedderDelegate>
+
+@end
+```
+
+4. Implement **NativeScriptEmbedderDelegate** protocol's **presentNativeScriptApp:** method:
```

Comment:
We've missed documenting the step to register the delegate (`[NativeScriptEmbedder sharedInstance].delegate = 
 ...`)